skimage/transform/tests/test_distance_tranform.py

```diff
@@ -0,0 +1,32 @@
+from skimage.transform.distance_transform import generalized_distance_transform as md
+import numpy as np
+
+def test_1d():
+    case = [1, 0, 0, 0, 1, 1, 0, 1, 1, 0, 0, 0, 0, 1, 0, 1, 1, 0, 1, 0]
+    out = [1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 1.0, 0.0, 1.0, 0.0]
+    if np.array_equal(md(np.asarray(case)),np.asarray(out)):

```

Comment:
You should replace this (and all the lines under it in this function) with `np.testing.assert_allclose`.

Reply:
Same for the tests below.